primefaces/src/main/java/org/primefaces/clientwindow/PrimeClientWindow.java

```diff
@@ -84,8 +85,6 @@ public void decode(FacesContext context) {
                 catch (IOException e) {
                     throw new FacesException("Could not send initial redirect!", e);
                 }
-
-                context.responseComplete();
```

Comment:
Had to remove this as its no longer part of LifeCycle and it kept reporting "response is closed".